portal/src/main.rs

```diff
@@ -49,7 +49,10 @@ async fn send_secret_to_app(app_id: &AppID, fd: std::os::fd::OwnedFd) -> Result<
     let collection = match service.default_collection().await {
         Err(oo7::dbus::Error::NotFound(_)) => {
             service
-                .create_collection("Default", Some(oo7::dbus::DEFAULT_COLLECTION))
+                .create_collection(
+                    oo7::dbus::DEFAULT_COLLECTION_LABEL,
+                    Some(oo7::dbus::DEFAULT_COLLECTION),
+                )
```

Comment:
instead, maybe we should have a `create_default` helper?

Reply:
I think that is fine to expose under unstable, make it easier / simpler to fix things.